vue-components/src/components/Lookup.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+	<label :class="[ 'wikit', 'wikit-Lookup', `wikit-Lookup--${width}` ]">
+		<div class="wikit-Lookup__label">{{ label }}</div>
+		<Input
+			@input.native="inputValue = $event.target.value"
+			@focus.native="focused = true"
```

Comment:
I might be missing something here, but is it really necessary to reimplement the native focus and blur with a controlled `focus` prop? I don't see it being used elsewhere within the component behaviour.